nixos/modules/hardware/hid-fanatecff.nix

```
@@ -0,0 +1,34 @@
+{ pkgs, lib, config, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.hid-fanatecff;
+  kernelPackages = config.boot.kernelPackages;
+in {
+  options.hardware.hid-fanatecff = {
+    enable = mkOption {
```

```suggestion
    enable = lib.mkEnableOption ''
      Linux module drivers for Fanatec wheel bases.
      Works with the CSL Elite and CSL/ClubSport DD/DD Pro,
      and has experimental support for the ClubSport V2/V2.5,
      Podium DD1/DD2 and CSR Elite.
    '';
```

I recommend you to use mkEnableOption for enable attributes.